masonite/request.py

```diff
@@ -241,19 +243,6 @@ def has(self, *args):
 
         return all((arg in self.request_variables) for arg in args)
 
-    def status(self, status):
```

Comment:
This should be added back

Reply:
@josephmancuso I have made the commit, 

Even while implementing I thought it is weird, I thought helper in another way.

Reply:
Where should I put `KeyError` exception?